source/classes.tex

```diff
@@ -2090,7 +2108,7 @@
 \indextext{special member function|see{destructor}}%
 
 \pnum
-In a declaration of a destructor, the \grammarterm{declarator} is a
+In a declaration of a prospective destructor, the \grammarterm{declarator} is a
```

Comment:
Add ref to [class.dtor] please.

Reply:
This is the first paragraph of [class.dtor].